cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -40,6 +41,9 @@ import (
 	"sigs.k8s.io/kueue/cmd/experimental/kjobctl/pkg/constants"
 )
 
+//go:embed templates/*
+var templates embed.FS
+
```

Comment:
nit: should this be in the slurm_builder file?

Reply:
Yeah, I thought about it. But the name is `templates` and it can work with another builders. Maybe we will use it for another modes. WDYT?

Reply:
> But the name is templates 
we can also change the name, I don't know about the future but for now it is only used in slurm. If/When it's needed in other builders we can think of moving it back along with the template parsing logic.

Reply:
Done